2023-03-10-3mdeb-msi-pro-z690-a-ddr4-first-qubes-certified-desktop.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: post
+title: "The 3mdeb MSI PRO Z690-A DDR4 is the first Qubes-certified desktop computer!"
+categories: announcements
+image: /attachment/posts/3mdeb-msi-pro-z690-a-ddr4_2.jpg
+---
+
+It is our pleasure to announce that the [3mdeb MSI PRO Z690-A DDR4](https://3mdeb.com/shop/open-source-hardware/dasharo-compatible-with-msi-pro-z-690a-ddr4-full-pc-build/) has become the fourth [Qubes-certified computer](/doc/certified-hardware/) for Qubes 4.X and the **first** Qubes-certified desktop computer **ever**!
+
+## About the 3mdeb MSI PRO Z690-A DDR4
+
+[![Photo of MSI PRO Z690-A DDR4 motherboard](/attachment/posts/3mdeb-msi-pro-z690-a-ddr4_1.jpg)](https://3mdeb.com/shop/open-source-hardware/dasharo-compatible-with-msi-pro-z-690a-ddr4-full-pc-build/)
+
+The [3mdeb MSI PRO Z690-A DDR4](https://3mdeb.com/shop/open-source-hardware/dasharo-compatible-with-msi-pro-z-690a-ddr4-full-pc-build/) is a full desktop PC build that brings the [Dasharo](https://dasharo.com/) open-source firmware distribution to the MSI PRO Z690-A DDR4 motherboard. The full configuration includes:
+
+| Part         | Model Name                                                      |
+|------------- | --------------------------------------------------------------- |
+| CPU	         | Intel Core i5-12600K, 3.7GHz                                    |
+| Cooling	     | Noctua CPU NH-U12S Redux (w/ Noctua NM-i17xx-MP78 Mounting Kit) |
```

Comment:
When I assembled my machine, the Noctua radiator and fan already came with a mounting kit, so a separate mounting is unnecessary. I will confirm it with our logistics specialist and update the shop description accordingly.

Reply:
Confirmed.  The mounting kit is obsolete and can be removed. Changes will also be made to our shop.

```suggestion
| Cooling	     | Noctua CPU NH-U12S Redux |
```